src/components/NavBar.vue

```diff
@@ -1,31 +1,113 @@
 <template>
-	<nav>
+	<div id="nav-bar">
 		<img src="../assets/logo.svg">
-		<a href="#about">About</a>
-		<a href="#speakers">Speakers</a>
-		<a href="#testimonials">Testimonials</a>
-		<a href="#sponsors">Sponsors</a>
-		<a href="#faq">FAQ</a>
-		<!-- <router-link to='/'>Home</router-link> -->
-		<!-- <router-link :to='{name: "Blog", params: {post: "main"}}'>Read our blog</router-link> -->
-	</nav>
+
+		<label v-if="isMobile" @click="openMenu">&#9776;</label>
+		<nav v-else>
+			<a href="#about">ABOUT</a>
+			<a href="#speakers">SPEAKERS</a>
+			<a href="#testimonials">TESTIMONIALS</a>
+			<a href="#sponsors">SPONSORS</a>
+			<a href="#faq">FAQ</a>
+		</nav>
+
+		<!-- MOBILE MENU -->
+		<div v-if="isMobile && isMenuOpen" class="mobileMenu">
```

Comment:
`v-if="isMobile"`

Holy shit Vue is incredibly powerful. The amount of media query magic I've had to do for this before... Love it.